Entity/AddressEmbeddable.php

```diff
@@ -13,67 +13,69 @@ class AddressEmbeddable extends Address
     /**
      * @ORM\Column(type="string", nullable=true)
      */
-    protected $locale;
+    protected string $locale = 'und';
 
     /**
      * @ORM\Column(type="string", nullable=true)
      */
-    protected $givenName;
+    protected string $givenName = '';
 
     /**
      * @ORM\Column(type="string", nullable=true)
      */
-    protected $additionalName;
+    protected string $additionalName = '';
 
     /**
      * @ORM\Column(type="string", nullable=true)
      */
-    protected $familyName;
+    protected string $familyName = '';
 
     /**
      * @ORM\Column(type="string", nullable=true)
      */
-    protected $organization;
+    protected string $organization = '';
 
     /**
      * @ORM\Column(type="string", nullable=true)
      */
-    protected $addressLine1;
+    protected string $addressLine1 = '';
 
     /**
      * @ORM\Column(type="string", nullable=true)
      */
-    protected $addressLine2;
+    protected string $addressLine2 = '';
+
+    protected string $addressLine3 = '';
```

Comment:
Why not add the orm annotation here? Shouldn't we store this in the database?

Reply:
This property was skipped until now. The value initiation was added to avoid errors while hydrating the object. It is the closest solution to the original bundle implementation. If you wish to add ORM mapping here, I don't mind. It's your bundle :)

Reply:
It was added in 2.x, so since that's the min requirement now, let's add it fully. See https://github.com/commerceguys/addressing/commit/5db4c30b171a374c3d1975e9d7f03518258d0156

Are there other properties that were added? (I'm guessing no since there'd be issues with hydrating otherwise).

Reply:
ok, you want it, you have it :)

Reply:
We'll just also need a setter, unless I'm looking over one.